### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by spidgorny**
Fix admin/check/ link

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Installation
 ------------
 
  * Extract the tarball into a location readable by your web server
- * Point your browser to http://path/to/mantisbt/admin/check.php to ensure that
+ * Point your browser to http://path/to/mantisbt/admin/check/ to ensure that
```
Comment (atrol) [inline readme.md]: Should be
```
* Point your browser to http://path/to/mantisbt/admin/check/index.php
```
as we should not depend on web server configuration.

**Comment by vboctor** [comment]
thanks @spidgorny :+1:

**Comment by dregad** [comment]
I'll merge this, thanks @spidgorny 

As a side note, I'm wondering if we would not be better off removing all installation instructions from the Readme file, and refer people to the Admin Guide instead, to avoid duplicating information...

**Comment by dregad** [comment]
Fixed in f2a8172cfebaa5a3ad6f2116d1cb16c1e6dcbab5

**Pull request closed without merging**